esx_service/cli/vmdkops_admin.xml

```diff
@@ -0,0 +1,571 @@
+        </command>
+        <!-- Policy commands -->
+        <command path="vdvs.policy.create">
+            <description>Create a storage policy</description>
+            <input-spec>
+                <parameter name="name" type="string" required="true">
+                    <description>The name of the policy</description>
+                </parameter>
+                <parameter name="content" type="string" required="true">
+                    <description>The VSAN policy string</description>
+                </parameter>
+            </input-spec>
+            <output-spec>
+                <string/>
+            </output-spec>
+            <format-parameters>
+                <formatter>simple</formatter>
+            </format-parameters>
+            <execute>/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py --output-format=xml policy create --name=$val{name} --content='$val{content}'</execute>
+        </command>
+        <command path="vdvs.policy.rm">
+            <description>'Remove a storage policy'</description>
+            <input-spec>
+                <parameter name="name" type="string" required="true">
+                    <description>The name of the policy</description>
+                </parameter>
+            </input-spec>
+            <output-spec>
+                <string/>
+            </output-spec>
+            <format-parameters>
+                <formatter>simple</formatter>
+            </format-parameters>
+            <execute>/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py --output-format=xml policy rm --name=$val{name}</execute>
+        </command>
+        <command path="vdvs.policy.ls">
+            <description>List storage policies and volumes using those policies</description>
+            <input-spec></input-spec>
+            <output-spec>
+                <list type="structure">
+                    <structure typeName="vdvs">
+                        <field name="Policy Name">
+                            <string/>
+                        </field>
+                        <field name="Policy Content">
+                            <string/>
+                        </field>
+                        <field name="Active">
+                            <string/>
+                        </field>
+                    </structure>
+                </list>
+            </output-spec>
+            <has-updates value="true"/>
+            <format-parameters>
+                <formatter>table</formatter>
+                <format-parameter name="fields:vdvs">Policy Name,Policy Content,Active</format-parameter>
+            </format-parameters>
+            <execute>/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py --output-format=xml policy ls</execute>
+        </command>
+        <command path="vdvs.policy.update">
+            <description>'Update the definition of a storage policy and all VSAN objects using that policy'</description>
```

Comment:
What do you mean "vdvs.policy.update" also update all VSAN objects using that policy?